.jenkins/perf_test/db_schema.py

```diff
@@ -0,0 +1,22 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import *
+
+Base = declarative_base()
+
+
+class PerfTestBaseline(Base):
+    __tablename__ = 'perf_test_baseline'
+
+    id = Column(String, primary_key=True)
+    commit_id = Column(String)
+    test_name = Column(String)
+    test_type = Column(String)
+    mean = Column(Float)
+    sigma = Column(Float)
```

Comment:
If we're going the database route, should we also record the individual experiment numbers?